src/super_gradients/recipes/coco2017_yolo_nas_m.yaml

```diff
@@ -0,0 +1,43 @@
+# YoloNAS-S Detection training on COCO2017 Dataset:
```

Comment:
Why do we need those in this PR ?

Reply:
These are temporary recipes to run the benchmarks on those models. Will be cleaned up when PR is finalized